src/Symfony/Component/Process/Pipes/WindowsPipes.php

```diff
@@ -163,14 +163,11 @@ public function close()
     /**
      * Creates a new WindowsPipes instance.
      *
-     * @param Process $process The process
-     * @param $input
-     *
      * @return WindowsPipes
      */
-    public static function create(Process $process, $input)
+    public static function create(Process $process, $input, $haveReadSupport)
```

Comment:
But we should now remove the `$process` parameter as it is not needed anymore, right?